src/document.rs

```diff
@@ -58,13 +58,13 @@ fn value_to_py(py: Python, value: &Value) -> PyResult<PyObject> {
         }
         Value::Date(d) => PyDateTime::new(
             py,
-            d.year(),
-            d.month() as u8,
-            d.day() as u8,
-            d.hour() as u8,
-            d.minute() as u8,
-            d.second() as u8,
-            d.timestamp_subsec_micros(),
+            d.into_utc().year(),
+            d.into_utc().month() as u8,
+            d.into_utc().day() as u8,
+            d.into_utc().hour() as u8,
+            d.into_utc().minute() as u8,
+            d.into_utc().second() as u8,
+            d.into_utc().microsecond() as u32,
```

Comment:
It is probably easier here to have once `into_utc()` call and then work on that result for all the values rather than calling it every time.

Reply:
Done